diff --git a/lib/stm32/f4/Makefile b/lib/stm32/f4/Makefile
index 3f59a1eb..0e8ce221 100644
--- a/lib/stm32/f4/Makefile
+++ b/lib/stm32/f4/Makefile
@@ -22,8 +22,9 @@ LIBNAME		= libopencm3_stm32f4
 SRCLIBDIR	?= ../..
 
 FP_FLAGS	?= -mfloat-abi=hard -mfpu=fpv4-sp-d16
-CC		= $(PREFIX)gcc
-AR		= $(PREFIX)ar
+CC = clang -target $(PREFIX)
+CFLAGS += --sysroot=/opt/local/cross-tool-cortex-m4-gcc13/arm-none-eabi/
+AR		= llvm-ar
 TGT_CFLAGS	= -Os \
 		  -Wall -Wextra -Wimplicit-function-declaration \
 		  -Wredundant-decls -Wmissing-prototypes -Wstrict-prototypes \
